content/_includes/embed.macros.njk

```diff
@@ -339,8 +311,10 @@ params:
   {%- set width = width or config.srcset | max -%}
   {%- for size in config.srcset -%}
     {%- if size <= width -%}
-      {%- if not loop.first %}, {% endif -%}
-      {{- src | imgSuffix(size) }} {{ size + 'w' -}}
+      {%- if not loop.first %},
+      {% endif -%}
+      {{- src | imgSuffix(size) }}
+      {{ size + 'w' -}}
```

Comment:
Are all of these changes auto-generated by a local code formatter? Some of them are breaking icons...

![Screen Shot 2020-12-05 at 6 45 51 AM](https://user-images.githubusercontent.com/552316/101241598-8bfcf480-36c5-11eb-8a21-532df648bc57.png)

Reply:
@mirisuzanne I'm curious about these formatting changes in `embed`... were those intentional?

Reply:
Yeah, I was struggling to get my new setup to not do this.

But… also that shouldn't happen. We disabled the indented code block markdown thing. 🧐

Anyway, thanks for fixing it.